src/popup/components/TransactionTagList.vue

```diff
@@ -110,7 +110,10 @@ export default defineComponent({
             ? t('transaction.dexType.pool')
             : t('common.swap'),
         );
-      } else if (props.transaction.claim) {
+      } else if (
+        props.transaction.claim
+        || innerTx.value.function === TX_FUNCTIONS.claim
```

Comment:
Unfortunately, any contract can have a method named claim. This method is not completely correct.

Reply:
Correct, I've fixed this by also checking for the contract address